.chloggen/pprofile-align-signal.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: pdata/pprofile
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Sync pdata/pprofile with OTel Profiling signal specification
```

Comment:
It would make more sense to describe the actual change here. That's the removed field (the reorder has no user impact)

Reply:
Reworded with https://github.com/open-telemetry/opentelemetry-collector/pull/13315/commits/c02a01dccb1e118b681f7357a27dc3f9ed32450f.